mod-inventory/src/main/resources/domain/mod-inventory/features/inventoryFeatureTest.feature

```diff
@@ -5,20 +5,48 @@ Feature: inventory sample
     * callonce login testUser
     * configure headers = { 'x-okapi-tenant':'#(testTenant)','Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': 'application/json, text/plain' }
 
-
-    Scenario: new Instance creation
-    * def createInstanceRequest = read('samples/createInstance.json')
+    Scenario: new Instance, Holdings, Item creation
+      * def testInstanceId = callonce uuid
       Given path 'inventory/instances'
-      And request createInstanceRequest
+      And request
+      """
+      {
+        "source":"FOLIO",
+        "title":"TestInstance",
+        "id":"#(testInstanceId)",
```

Comment:
It should be possible to remove this entirely, the server should generate an ID if one is not provided.